app/models/host.rb

```diff
@@ -908,7 +908,7 @@ def reset_discoverable_fields
   def detect_discovered_os(ost)
     # Determine os
     os_type = nil
-    if vmm_vendor == "vmware"
+    if vmm_vendor_display == "vmware" # ??? This will never be true
```

Comment:
@Fryguy :+1: I will use that method if that's right....just concerned that AFAICS that path wasn't executing before, so wondered if was still needed.